communote/webapp/src/main/webapp/WEB-INF/vm/widget/blog/CreateNote.Widget.html.vm

```diff
@@ -80,17 +78,16 @@
                     <input id="${widget.WidgetId}-tag-search" type="text" placeholder="$fmtTool.message('blogpost.create.tags.hint')"/>
                 </div>
             </div>
-            <div class="cn-write-note-accessory-attachment cn-accessory-container#if($widget.getBooleanParameter('customUpload', false)) cn-custom-upload#end" style="display:none;">
+            <div class="cn-write-note-accessory-attachment cn-accessory-container" style="display:none;">
                 <div class="cn-accessory-label"><span class="cn-icon"><!-- Empty --></span><span class="cn-icon-label">$fmtTool.message('blogpost.create.tab.attachment.title')</span></div>
                 <div class="cn-accessory-content">
-                    <div id="${widget.WidgetId}-summary-attachments" class="cn-attachment-summary cn-border cn-hidden"></div>
-
+                    <div id="${widget.WidgetId}-attachments-summary" class="cn-attachment-summary cn-border cn-hidden"></div>
+                    <a href="javascript:;" class="cn-button" id="${widget.WidgetId}-attachments-upload-button">$fmtTool.message('widget.createNote.attachments.uploadButton')</a>
```

Comment:
@rwi Auch eher als Idee:

* href="javascript:;" kann man sich ja eigentlich sparen. Lieber per CSS den korrekten Mauszeiger anzeigen
* Statt auf der id des Elements zu arbeiten sollte das generischer sein, damit man die Action auf mehrere Elemente legen kann
  * idealerweise ist das Widget ja irgendwo gekapselt von der id des Widgets, so dass man innerhalb des Containers mit CSS-Klassen oder Datenattributen arbeiten könnte, also in etwas so:

<div id="${widget.WidgetId}-wrapper">
   ...
   <a class="cn-button action-upload-attachment">...</a>
   <a class="cn-button" data-action="upload-attachment">...</a>
   ...
</div>

Reply:
richtig, hatte ich in dem vorherigen Stand, bei dem es den Button nicht mehr gab, auch so ähnlich realisiert. Beim Wiedereinbauen des Buttons habe ich dann den Weg des geringsten Widerstands gewählt :wink: